src/main/java/core/basesyntax/service/impl/FileReaderImpl.java

```diff
@@ -0,0 +1,20 @@
+package core.basesyntax.service.impl;
+
+import core.basesyntax.service.FileReaderService;
+import java.io.IOException;
+import java.nio.file.Files;
+import java.nio.file.Path;
+import java.util.List;
+
+public class FileReaderImpl implements FileReaderService {
+    @Override
+    public List<String> readTheFruitsStorage(String pathName) {
```

Comment:
Let's rename this method to make it more reusable. If this method only read data we don't need to mention fruits here